src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -225,26 +239,30 @@ export default class CaseAlterationTable extends React.Component<ICaseAlteration
                     getPseudoOqlSummaryByAlterationTypes(data.oqlDataByGene, oql.gene, alterationTypes)!.summaryContent.toUpperCase().includes(filterStringUpper),
                 visible: false
             });
-            //add column for each gene alteration combination
-            alterationTypes.forEach(alterationType => {
-                columns.push({
-                    name: `${oql.gene} ${alterationType}`,
-                    tooltip: <span>{oql.oql_line}</span>,
-                    headerDownload: (name: string) => `${oql.gene} ${alterationType}`,
-                    render: (data: ICaseAlteration) => {
-                        const pseudoOqlSummary = generatePseudoOqlSummary(data.oqlDataByGene, oql.gene, alterationType); 
+        });
+
+        this.props.oqls.forEach(oql => {
+            //add column for each oql
+            const alterationTypes = oql.parsed_oql_line.alterations ? 
+                _.uniq(_.map(oql.parsed_oql_line.alterations, (alteration) => {
+                    return alteration.alteration_type.toUpperCase();
```

Comment:
why uppercase?  if it's for printing, maybe it could be done at time of print

Reply:
This uppercase value will be used in the following file, do you want me to uniform the string at here? : https://github.com/dippindots/cbioportal-frontend/blob/d8710930fc61576ed54d0f1a3373a7023380a681/src/pages/resultsView/download/CaseAlterationTable.tsx#L79